cpp/WKTJsiWorkletContext.h

```diff
@@ -92,15 +92,24 @@ class JsiWorkletContext
                   std::function<void(std::function<void()> &&)> jsCallInvoker);
 
   /**
-   Static / singleton default context
+   Get the default context
    */
-  static std::shared_ptr<JsiWorkletContext> getDefaultInstance() {
+  static std::shared_ptr<JsiWorkletContext> getDefaultInstanceAsShared() {
     if (defaultInstance == nullptr) {
       defaultInstance = std::make_shared<JsiWorkletContext>();
     }
     return defaultInstance;
   }
 
+/**
+ * @brief Get the Default Instance object
+ * 
+ * @return JsiWorkletContext* 
+ */
+  static JsiWorkletContext *getDefaultInstance() {
+    return JsiWorkletContext::getDefaultInstanceAsShared().get();
+  }
```

Comment:
Do we need this? I feel like using `shared_ptr` everywhere makes things a bit safer?

Reply:
Absolutely - We make sure that when we pass it out of the context we convert it to a smart pointer - using it as a pointer is only internal to the Context class so it should be 100% safe.